cpp/src/arrow/integration/meson.build

```diff
@@ -0,0 +1,29 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+install_headers(['json_integration.h'])
+
+exc = executable(
+    'arrow-json-integration-test',
+    sources: ['json_integration_test.cc'],
+    dependencies: [arrow_dep, rapidjson_dep, gflags_dep, gtest_dep],
+    link_with: [arrow_test_lib],
+)
+
+if needs_tests
```

Comment:
Can we remove this?

Reply:
In this particular case I don't think so - the CMake configuration compiles `json_integration_test.cc` into an executable, but only makes it a test if `ARROW_BUILDS_TESTS` is true, leaving it as a standalone executable otherwise.

Reply:
Ah, I see. We use the executable for integration test too.